Bugsnag/Configuration/BugsnagConfiguration.m

```diff
@@ -343,7 +344,7 @@ - (void)removeOnBreadcrumbBlock:(BugsnagOnBreadcrumbBlock _Nonnull)block {
 - (NSDictionary *)errorApiHeaders {
     return @{
              kHeaderApiPayloadVersion: @"4.0",
-             kHeaderApiKey: self.apiKey,
```

Comment:
This should either stay as-is or we should move the other constants to `BugsnagKeys.h`. I don't see any wider use to make this worthwhile moving.

Reply:
Moved constants to `BugsnagKeys`.